html-formatter/nodejs/lib/html_stream.jsx

```diff
@@ -1,7 +1,8 @@
 import Stream from "stream"
 import React from "react"
 import ReactDOMServer from "react-dom/server"
-import {Cucumber} from "./cucumber_react/cucumber_react"
+import {CucumberReact} from "cucumber-react"
+const {Cucumber} = CucumberReact
```

Comment:
`import {Cucumber as CucumberReact} from "cucumber-react"`? This `const` trick is done in cucumber-react tests and possibly other places as well. It looks a bit odd.